### DISCUSSION TITLE
Where does my router syslog get sent to in SecurityOnion?

### DISCUSSION THREAD

**Discussion by tanc7** [Security-Onion-Solutions/securityonion#10008] [Unsupported Versions]
I configured my router to send it's syslog over the LAN to my SecurityOnion node listening on port 514. I seen it working but I don't know where this information is stored.

https://docs.securityonion.net/en/2.3/syslog.html

![image](https://user-images.githubusercontent.com/26513830/227823679-f47dead0-d6b6-44ff-b90c-31f5ac384d38.png)

I would like to parse through them so I can report on the IP addresses scanning my public IP by looking at it's firewall notifications. I already can do this and make a formatted csv file.

**Comment by cm-ops** [comment]
One the data goes through the ingest pipeline it will be in an index `so-syslog-*` you can run this command from the CLI - `so-elasticsearch-query _cat/indices | grep syslog` to see the indices. You would run that on your node that is performing search node functions.

**Comment by kwwv** [comment]
@tanc7 fwiw you can also use `metadata.pipeline:syslog` in kibana discovery to view data that has passed through the syslog pipeline.
